helm/chart/router/values.yaml

```diff
@@ -218,6 +218,15 @@ autoscaling:
   #     type: cpu
   #     targetUtilizationPercentage: 75
 
+
+rollingUpdate:
+  {}
+# Adjust rolling update strategy.  Can take absolute values or % values.
+# (https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#rolling-update-deployment)
+# Defaults if not set are:
+#  maxUnavailable: 25%
+#  maxSurge: 25%
+
```

Comment:
Would be nice to have line 224 moved to line 221 and specified as a helm-docs style comment

Reply:
I've adjusted and used helm-doc style comment

Reply:
@garypen - are you good with this PR now with the adjustment made?

Reply:
@garypen -- Sorry for the harassment, but can we get this merged in.  I have at least one more PR I'd like to submit, but I'd like this one settled first before I do so.

Reply:
@theJC Sorry about the delay, I've been on vacation.